packages/api/src/modules/config/config.ts

```diff
@@ -24,11 +24,25 @@ export class Config {
     port: number | undefined;
   };
 
-  authSecrets: {
-    cookie: string | undefined;
-    access: string | undefined;
-    refresh: string | undefined;
-    logout: string | undefined;
+  cookie: {
+    secret: string | undefined;
+  };
+
+  tokens: {
+    accessToken: {
+      name: string;
```

Comment:
It shouldn't be rather as constants with auth tokens name? What case it could wrap to change it from envs?